includes/admin/downloads/metabox.php

```diff
@@ -1126,17 +1168,15 @@ function edd_render_accounting_options( $post_id ) {
 	$edd_sku = get_post_meta( $post_id, 'edd_sku', true ); ?>
 
 	<div class="edd-product-options-wrapper">
-		<p><strong><?php _e( 'Accounting Options', 'easy-digital-downloads' ); ?><span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php _e( '<strong>SKU</strong>: If a SKU is entered for this product, it will be shown on the purchase receipt and exported purchase histories.', 'easy-digital-downloads' ); ?>"></span></strong></p>
-		<p>
-			<label>
-				<?php echo EDD()->html->text( array(
-					'name'  => 'edd_sku',
-					'value' => $edd_sku,
-					'class' => 'small-text'
-				) ); ?>
-				<?php echo sprintf( __( 'Enter an SKU for this %s.', 'easy-digital-downloads' ), strtolower( edd_get_label_singular() ) ); ?>
-			</label>
-		</p>
+		<div class="edd-product-options__title"><?php _e( 'Accounting Options', 'easy-digital-downloads' ); ?><span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php _e( '<strong>SKU</strong>: If a SKU is entered for this product, it will be shown on the purchase receipt and exported purchase histories.', 'easy-digital-downloads' ); ?>"></span></div>
+		<label>
```

Comment:
All of the inputs in the sidebar have more unique/customized markup already and I'm not completely sure how to tackle these in a way that's still backwards compatible. Should the `edd-product-options-wrapper` just also be the `edd-form-group`, do you think? Some of the `edd-product-options__title` are `div` and some are `label`. And some of the labels, like this one, are wrapped, so don't _technically_ need the `for`, but yes, I think they would be better with them. Just wasn't quite sure how to approach the sidebar.

Reply:
@LisaCee what do you think about this?

```php
	<div class="edd-form-group edd-product-options-wrapper">
		<div class="edd-product-options__title"><?php esc_html_e( 'Accounting Options', 'easy-digital-downloads' ); ?><span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php _e( '<strong>SKU</strong>: If a SKU is entered for this product, it will be shown on the purchase receipt and exported purchase histories.', 'easy-digital-downloads' ); ?>"></span></div>
			<div class="edd-form-group__control">
				<label class="edd-form-group__label" for="edd_sku">
					<?php printf( esc_html__( 'Enter an SKU for this %s.', 'easy-digital-downloads' ), strtolower( edd_get_label_singular() ) ); ?>
				</label>
				<?php echo EDD()->html->text(
					array(
						'name'  => 'edd_sku',
						'id'    => 'edd_sku',
						'value' => $edd_sku,
						'class' => 'edd-form-group__input small-text',
					)
				);
				?>
		</div>
	</div>
```
It does reorder the label and input visually, but other than that, no visual changes, just the updated markup. (Note: that `strtolower` should be removed for translation purposes.)

Reply:
I like this a lot.  I actually prefer the label coming before the input field.